TenureInformationApi/V1/Controllers/TenureInformationController.cs

```diff
@@ -74,12 +82,27 @@ public async Task<IActionResult> PostNewTenure([FromBody] CreateTenureRequestObj
         [LogCall(LogLevel.Information)]
         public async Task<IActionResult> UpdateTenureForPerson([FromRoute] UpdateTenureRequest query, [FromBody] UpdateTenureForPersonRequestObject updateTenureRequestObject)
         {
-            var token = _tokenFactory.Create(_contextWrapper.GetContextRequestHeaders(HttpContext));
+            var contextHeaders = _contextWrapper.GetContextRequestHeaders(HttpContext);
+            var token = _tokenFactory.Create(contextHeaders);
 
-            var tenure = await _updateTenureForPersonUseCase.ExecuteAsync(query, updateTenureRequestObject, token).ConfigureAwait(false);
-            if (tenure == null) return NotFound(query.Id);
+            string ifMatchString = contextHeaders.GetHeaderValue(HeaderConstants.IfMatch);
+            var ifMatch = int.TryParse(ifMatchString, out int i) ? i : (int?) null;
```

Comment:
I don't think this will work - the header will format `"0"` so you'll need to remove the quotes and then parse it to an int.

Reply:
I was wondering whether the value would be returned as a quoted string. - I guess it will then.

Reply:
I've added code to remove the quotes